### PR TITLE
Bugfix/manual maintenance release project wide

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/models/package.rb [hunk 1]
```diff
@@ -1400,6 +1400,11 @@ def send_sysrq(params)
     backend_build_command(:sendsysrq, params[:project], params.slice(:package, :arch, :repository, :sysrq))
   end
 
+  def target_name
+    # The maintenance ID is always the sub project name of the maintenance project
+    project.is_maintenance_incident? ? "#{name}.#{project.basename}" : name
+  end
+
   def release_target_name
```
Comment (bgeuken) [inline src/api/app/models/package.rb]: @adrianschroeter Do you know if there is any difference between those two methods (except that release_target_name also covers old incidents)?

**Comment by ChrisBr** [comment]
@bgeuken please rebase! Also what is the state of this PR? Do we need to wait for feedback from @adrianschroeter ?

**Comment by bgeuken** [comment]
> @ChrisBr commented 7 days ago
> @bgeuken please rebase! Also what is the state of this PR?

It needs a review. Hence the PR :stuck_out_tongue: 

> Do we need to wait for feedback from @adrianschroeter ?

No. The change doesn't cause any regression and we can investigate later on whether we can merge these two methods or not.

**Comment by bgeuken** [comment]
Meanwhile I found a way to replace the mock :bowtie:

**Comment by bgeuken** [comment]
No idea why this fails in Circle CI. The error message sounds like a VCR cassette is missing, but this does not happen in Travis or on my local system.

**Comment by ChrisBr** [comment]
@bgeuken could you try to rebase?

**Comment by ChrisBr** [comment]
> Okay, found the problem

Are you sharing it with us :pray: :smile:

**Comment by bgeuken** [comment]
:smile: 
The user name was not static. So it the cassettes wouldn't match when the tests were run in different order. 
Not sure why this wasn't a problem for travis though.

**Pull request merged**